purchase_work_acceptance/views/work_acceptance_views.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_work_acceptance_form" model="ir.ui.view">
+        <field name="name">work.acceptance.form</field>
+        <field name="model">work.acceptance</field>
+        <field name="arch" type="xml">
+            <form string="Work Acceptance">
+               <header>
+                   <button name="button_accept" type="object" states="draft" string="Accept" class="oe_highlight"/>
+                   <button name="button_draft" type="object" states="accept,cancel" string="Set to draft"/>
+                   <button name="button_cancel" type="object" states="draft,accept" string="Cancel"/>
+                   <field name="state" widget="statusbar" statusbar_visible="draft,accept" readonly="1"/>
+               </header>
+               <sheet>
+                   <div class="oe_title">
+                       <h1><field name="name" readonly="1"/></h1>
+                   </div>
+                   <group>
+                       <group>
+                           <field name="partner_id"/>
+                           <field name="responsible_id"/>
+                           <field name="invoice_ref"/>
+                           <field name="currency_id" groups="base.group_multi_currency" options="{'no_create': True}"/>
+                       </group>
+                       <group>
+                           <field name="date_due"/>
+                           <field name="date_receive"/>
+                           <field name="date_accept" attrs="{'invisible': [('state', '=', 'draft')]}"/>
+                           <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                       </group>
+                   </group>
+                   <notebook>
+                       <page string="Products">
+                           <field name="wa_line_ids" attrs="{'readonly': [('state', '!=', 'draft')]}">
+                                <tree string="Work Acceptance Lines" editable="bottom">
```

Comment:
Do not allow adding more WA Lines (create="0"). Currently, we can, but when save, it throw error.

Reply:
Even WA Line in WA is deleted, create Vendor Bill still not remove that line.

Reply:
When create another WA, the WA for service should default with Quantity - Billed Qty.
Now, it still default with full amount.
Note: this will be different for storable product.

Reply:
Fix about Invoice reference too,
https://github.com/ecosoft-odoo/kanban/issues/11#issuecomment-550725825

Reply:
> Do not allow adding more WA Lines (create="0"). Currently, we can, but when save, it throw error.

I think it should allow adding more WA Lines for flexible.

> Even WA Line in WA is deleted, create Vendor Bill still not remove that line.
BUT...for IN it is ok not to remove.

Done

> When create another WA, the WA for service should default with Quantity - Billed Qty.
Now, it still default with full amount.
Note: this will be different for storable product.

Done